src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/Utils.java

```diff
@@ -43,4 +43,18 @@ public static String appendQueryString(String original, String queryStringValue)
             throw new UnsupportedOperationException("Cannot handle type class: " + type.getClass());
         }
     }
+    
+    @SuppressWarnings("unchecked")
+	public static <T> T cast(Class<?> returnClass, Object obj) {
```

Comment:
```suggestion
    public static <T> T cast(Class<?> returnClass, Object obj) {
```

Reply:
How did you notice this?

Reply:
![image](https://user-images.githubusercontent.com/7574801/92783375-9162d100-f35a-11ea-9986-70991c361f7f.png)

Reply:
Interesting. They show up as the same width for me. Did you somehow configure a tab width of 6 on github?

Reply:
Nope, and just looked with a different browser where I'm not logged in and it still shows a difference.

Reply:
Ah. It's my Refined GitHub extension that makes the tabwidth 4 spaces.

Reply:
I really need to configure Eclipse to treat tabs as spaces 😆